server/infrastructure/player_repository.py

```
@@ -0,0 +1,56 @@
+        """Gets a specific player from database through mlb id"""
+        if not self.db:
+            raise HTTPException(
+                status_code=status.HTTP_503_SERVICE_UNAVAILABLE,
+                detail="Firebase is not configured"
+            )
+        try:
+            player_document = self.db.collection('players').document(str(player_id)).get()
+            if not player_document.exists:
+                return None
+            return player_document.to_dict()
+        except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)